assignments/mgmt-groups/mg-platform-sandbox/assign.expiresaftertag.json

```
@@ -0,0 +1,26 @@
+{
+    "properties": {
+      "metadata": {
+        "assignedBy": "PlatOps"
+      },
+      "description": "Policy to check required tags",
+      "displayName": "Expires After Tagging - PLATFORM-SANDBOX",
```

all others are prefixed with HMCTS, what about this for a name?
```suggestion
      "displayName": "HMCTS Expires After Tagging - (mg:Platform-Sandbox)",
```